appveyor.yml

```diff
@@ -0,0 +1,26 @@
+build: off
+
+clone_folder: c:\gopath\src\gonum.org\v1\gonum
+
+cache:
+  - '%LocalAppData%\go-build'
+
+branches:
+  only:
+    - master
+
+environment:
```

Comment:
It's worth adding a note to not move these lines as exists in the [travis config](https://github.com/gonum/gonum/blob/921c8b9ec6c531bca1ab15b67c348869a53625bd/.travis.yml#L5-L6).